dashboard/db/migrate/20170403224005_add_first_activity_time_to_section.rb

```diff
@@ -0,0 +1,8 @@
+class AddFirstActivityTimeToSection < ActiveRecord::Migration[5.0]
+  def change
+    # Give existing rows a fake first_activity_at that's in the past, but
+    # newly created rows should default to nil
+    add_column :sections, :first_activity_at, :datetime, default: DateTime.parse('1st Jan 1970')
+    change_column_default :sections, :first_activity_at, from: DateTime.parse('1st Jan 1970'), to: nil
```

Comment:
Is there a compelling reason not to just start by defaulting the column value to `nil`? And, as necessary, back-populate the values quickly after the migration?